components/brave_ads/core/internal/database/database.cc

```diff
@@ -30,7 +30,9 @@
 
 namespace brave_ads {
 
-Database::Database(base::FilePath path) : db_path_(std::move(path)) {
+Database::Database(base::FilePath path)
+    : db_path_(std::move(path)),
+      db_(sql::Database::Tag("RewardsInternalDatabase")) {
```

Comment:
AdsInternalDatabase

Reply:
Fixed in [a90cd4a](https://github.com/brave/brave-core/pull/26537/commits/a90cd4a12c6523c6ea9dd1c513e8cc3d5284e225)